### ISSUE TITLE
Help in getting started

### ISSUE THREAD

**Issue by RameshRM** [uber/AthenaX#34]
Hi ,

`AthenaX` looks like a perfect fit for our usage, we are in the process of building a platform where in users can submit jobs , aggregate and perform sinks. Sadly the docs are not helping, can u share a sample docs which helps understand on how `kafka` works and sink with `es` Some of the docs are not relevant with `class files not found`

**Comment by thinkinerp** [comment]
I agree with RameshRM ，there are so few connecters and catalog impls that developer will rework with wheel. So I await to seen basic connectors and catalog ,like HBase ,kafka and so on , in release. thx.
